src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -22,6 +22,9 @@
  */
 class DateTimeToLocalizedStringTransformer extends BaseDateTimeTransformer
 {
+    // Midnight of last day of 9999 to prevent 5 digit years from being transformed
+    const MAX_TIMESTAMP = 253370674800;
```

Comment:
This seems to be the last day of the year 9998. Shouldn't this be `253402297199` (or at least `253402210800`)?

Reply:
Uhm you're right, I must've c/p'ed the wrong line in my tests haha. I did consciously go for UTC midnight of the 31st, as there are +13h timezones and this means it will never be the year 10000 at any (currently known) timezone for the given timestamp. The correct value would be 253402214400 which is `9999-12-31T00:00:00+00:00`.